pages/releases/4.3.html

```diff
@@ -0,0 +1,2313 @@
+										the small text found in user interfaces.
+									</p>
+									<p>
+										By adding your UI in an
+										<code class="highlight"><span class="basetype">OpenXRCompositionLayer</span></code>
+										node, you can force the XR compositor to directly render a viewport,
+										leading to crisper and clearer 2D content.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dsnopek,BastiaanOlij"
+									data-read-more="https://github.com/godotengine/godot/pull/89880"
+								></div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<video class="release-card-video lazy" playsinline disablepictureinpicture autoplay loop muted
+								poster="/storage/releases/4.3/images/composition-layers.webp">
+								<source data-src="/storage/releases/4.3/video/composition-layers.webm"
+									type="video/webm" />
+							</video>
+						</div>
+					</div>
+				</div> <!-- #xr-2d-content-renders-less-blurry -->
+
+				<div id="xr-standardized-hand-body-face-tracking" class="release-card">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-standardized-hand-body-face-tracking">
+										Standardized hand, body, and face tracking
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										In this release, we introduced a new standardized system for XR hand tracking,
+										which allows developers to use one kind of node (and code) to create a hand
+										tracking experience that will work in multiple XR eco-systems. This is currently
+										supported by both OpenXR and WebXR.
+									</p>
+									<p>
+										Brand new features are systems for body and face tracking, that are built in the
+										same standartized way, but currently only supported in OpenXR on Meta Quest headsets.
+									</p>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div> <!-- #xr-standardized-hand-body-face-tracking -->
+
+				<div id="xr-performance-stability-improvements" class="release-card span-3">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-performance-stability-improvements">
+										Performance and stability improvements
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Due to countless improvements made to this part of the engine, XR applications are now more stable and performant.
+									</p>
+									<p>
+										Highlights include:
+									</p>
+									<ul>
+										<li>
+											improved timing of tracking data
+										</li>
+										<li>
+											XR is now thread safe, allowing you to move rendering onto a separate thread
```

Comment:
This isn't _fully_ true. A lot of work was done on allowing multi-threaded rendering, and probably the XR bits are good, but due to other problems in multi-threaded rendering I'm not sure we can really recommend users actually do this in practice.